### PR TITLE
Update repeated.md

### PR DESCRIPTION
**Pull request by jtdendunnen**
fixed errors in the examples given, now matching with the long format

### CODE REVIEW

**Review by reece — APPROVED** [review]
Looks good to me!

File: docs/recommendations/DNA/repeated.md [hunk 1]
```diff
@@ -35,36 +35,39 @@ Format (mixed repeat):   **"prefix""range_repeated_sequence""repeat_sequence_uni
...
-            * NM\_002024.5:c.-129CGG[79]: a repeated CGG tri-nucleotide sequence starting at position c.-129 with 79 CGG copies: **NOTE**:    by definition, since the reference sequence contains a mixed repeat (CGG and AGG units), the variant description only covers the sequence up to the first AGG interruption (position c.-99). A better description, giving less confusion, is NM\_002024.5:c.-128\_-69GGC[78]GGA[1]GGC[10] (i.e. assuming only the length of the first CGG segment was meant and not the total length of the GGC-GGA-GGC repeat).
-            * NM\_002024.5:c.-128GGM[108]: a repeated mixed tri-nucleotide sequence starting at position c.-129 with 108 GGC/GGA copies
+            * NM\_002024.5:c.-128\_-69GGC[68]GGA[1]GGC[10]: a repeated CGG tri-nucleotide sequence starting at position c.-129 with 79 repeat units: **NOTE**:    since the reference sequence contains a mixed repeat (CGG and AGG units), the variant **can not** be described as NM\_002024.5:c.-129CGG[79]. NM\_002024.5:c.-129CGG[79] would cover only the sequence up to the first AGG interruption (position c.-99).
```
Comment (ifokkema) [inline docs/recommendations/DNA/repeated.md:58]: I'm confused; does the repeat start at `c.-129` or `c.-128`?

File: docs/recommendations/DNA/repeated.md [hunk 2]
```diff
@@ -35,36 +35,39 @@ Format (mixed repeat):   **"prefix""range_repeated_sequence""repeat_sequence_uni
...
         * **not sequenced**
-            * NC\_000003.12:g.(63912602\_63912844)insN[9] | NM\_000333.3:c.(4\_246)insN[9]: a fragment containing the AGC repeat in the ATXN7 gene was amplified (from nucleotide g.63912602/c.4 to g.63912844/c.246) and its size determined to be 9 nucleotides larger ( "insN[9]" ) then that of the reference sequence.: **NOTE**:    since the fragment was not sequenced the variant can not be described as g.63912687AGC[13] / c.89AGC[13].
+            * NC\_000003.12:g.(63912602\_63912844)insN[9] | NM\_000333.3:c.(4\_246)insN[9]: a fragment containing the AGC repeat in the ATXN7 gene was amplified (from nucleotide g.63912602/c.4 to g.63912844/c.246) and its size determined to be 9 nucleotides larger ( "insN[9]" ) compared to that of the reference sequence.: **NOTE**:    since the fragment was not sequenced the variant can not be described as g.63912687\7\_63912716AGC[13] / c.89\_118AGC[13].
```
[the remark this answers is not in the document]
Comment (ifokkema) [reply docs/recommendations/DNA/repeated.md:51]: Fixed in 953d5d73cb59644f77ac0004ecf3a1823ae9f6cf.

File: docs/recommendations/DNA/repeated.md [hunk 3]
```diff
@@ -35,36 +35,39 @@ Format (mixed repeat):   **"prefix""range_repeated_sequence""repeat_sequence_uni
...
 
-    A complex case. First note that by applying the <b>3'rule</b> it is a <b>variable GT and not TG stretch</b>. When the coding DNA reference sequence has TG11 followed by T7, the reference allele is described as c.1210-33_1210-6GT[11]T[6]. When only variability of the T-stretch is reported, the reference allele is described as c.1210-12T[7].<br>To indicate the overall variability found in the population the description is c.1210-33_1210-6GT[9_13]T[4_8] for the combined repeat and c.1210-12T[5_9] for the T-stretch.
+    A complex case. First note that by applying the <b>3'rule</b> it is a <b>variable GT and not TG stretch</b>. When the coding DNA reference sequence has TG11 followed by T7, the reference allele is described as c.1210-33_1210-6GT[11]T[6]. When only variability of the T-stretch is reported, the reference allele is described as c.1210-12T\_1210-6[7].<br>To indicate the overall variability found in the population the description is c.1210-33_1210-6GT[9_13]T[4_8] for the combined repeat and c.1210-12T[5_9] for the T-stretch.
```
[the remark this answers is not in the document]
Comment (ifokkema) [reply docs/recommendations/DNA/repeated.md:73]: Fixed in 953d5d73cb59644f77ac0004ecf3a1823ae9f6cf.

**Pull request merged**